CHANGELOG.md

```diff
@@ -1,12 +1,19 @@
+## v0.6.3
+
+-   [Enhancement] change app portal url from `/app-portal` to `/app` #924
+-   [Enhancement] App portal improments #918
```

Comment:
@jirevwe I think this should be`improvements`

Reply:
Doesn't enhancement mean improvement in this context?